libjpeg-turbo.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: libjpeg-turbo
-  version: 3.0.3
-  epoch: 1
+  version: 3.0.90
+  epoch: 0
```

Comment:
closing this one given this is a pre-release and the update block patch was merged here https://github.com/wolfi-dev/os/pull/28390 now we rely on GitHub releases for updates w.r.t. this package.